packages/ckeditor5-link/src/linkcommand.ts

```diff
@@ -458,7 +458,7 @@ function findChanges( oldText: string, newText: string ) {
  * @param linkRange Range of the entire link element
  * @returns Text node for content updates
  */
-function getLinkPartTextNode( range: Range, linkRange: Range ) {
+export function getLinkPartTextNode( range: Range, linkRange: Range ): Item | null {
```

Comment:
I'd mark this function as internal. As I understand it is exported only for test purposes.